packages/vite/types/importMeta.d.ts

```diff
@@ -54,7 +52,7 @@ interface ImportMeta {
 
   glob(
     pattern: string,
-    options?: AssertOptions
+    options?: import('../src/node/importGlob').AssertOptions
```

Comment:
FYI this breaks the production client type because this file is referenced in `vite/client.d.ts` and in production `src/node/importGlob.ts` doesn't exist.

For now we should probably just duplicate the type here instead.

Reply:
Done in https://github.com/vitejs/vite/pull/6531